routes/global_challenge.py

```diff
@@ -79,14 +94,20 @@ def get_community_saving():
     :rtype: dict or type(None) if there are no values
     """
 
-    query = "SELECT timestamp, saving FROM community_saving ORDER BY timestamp DESC"
+    # query = "SELECT timestamp, saving FROM community_saving ORDER BY timestamp DESC"
 
     try:
         engine = get_engine()
-        with engine.connect() as con:
+        with engine.connect():
 
             # Query last community saving prognosis
-            community_saving = con.execute(query).first()
+            query_result = []
+            session = create_session()
+            for row in session.query(CommunitySaving).order_by(CommunitySaving.timestamp.desc()):
+                query_result.append((row.timestamp, row.saving))
+
+            community_saving = query_result.first()
+
```

Comment:
Die Routen haben übrigens alle auch eigene Unit Tests. Da kannst Du Dir dann Ergebnisse ausgeben lassen und schauen, ob das, was zurückkommt, noch mit der API-Beschreibung in https://github.com/buzzn/broker-evaluation/blob/master/api.md übereinstimmt.